modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts

```diff
@@ -25,6 +25,7 @@ import {
 import {getWorkerURL, WorkerFarm} from '@loaders.gl/worker-utils';
 
 const I3S = 'I3S';
+const ERROR_MESSAGE = 'Tile converter does not work in browser, only in node js environment';
```

Comment:
```suggestion
const ERROR_MESSAGE = 'Tile converter does not work in browser, only in NodeJS environment';
```

Reply:
Move it to `constants.ts` file in the root of the module. We have the same message in 2 files.